umbraco/Umbraco-CMS
# Language variant deletion

Fixes: #6663 

Testing:
- Go to the **Settings** section and enable the toggle that will allow the **Home** Document type to vary by culture
- In the same section, add Danish language
- Go back to the **Content** section and fill in name of the Home node after you choose to see the tree in Danish - _Home DK_
- Click **Save**
- Choose to see the **Content** tree in Danish and notice that the selected language will update the `mculture` attribute in the url to `mculture=da`
- Navigate to **Setting** > **Languages** next and delete the Danish language, leaving you just with the default one
- Go back to the **Content** section and verify that your tree has been updated as well as the `mculture` (it would disappear if this is the last non-default language or it will be set to the default language if you have more non-default languages)
- Clicking on the Home node and saving without any changes will be successful
